adback-solution-to-adblock/class-ad-back.php

```diff
@@ -27,7 +27,7 @@ public function getContents($url)
         }
     }
 
-    public function postContents($url, $fields, $header = array())
+    public static function postContents($url, $fields, $header = array())
```

Comment:
You have a call in `saveMessage()` to `$this->postContents(...)`, you should update it to `self::postContents()` I think